### ISSUE TITLE
Wrong file permission on `lib/sass-rails.rb` @ v5.0.2 package

### ISSUE THREAD

**Issue by gussan** [rails/sass-rails#323]
`lib/sass-rails.rb` is not readable except for an installed user.

* 5.0.1

```bash
gussan ~$ ls -al ~/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/sass-rails-5.0.1/lib/sass-rails.rb
-rw-r--r--+ 1 gussan  staff  21  3 31 14:57 ~/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/sass-rails-5.0.1/lib/sass-rails.rb
```

* 5.0.2

```bash
gussan ~$ ls -al ~/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/sass-rails-5.0.2/lib/sass-rails.rb
-rw-------+ 1 gussan  staff  21  3 31 15:04 ~/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/sass-rails-5.0.2/lib/sass-rails.rb
```

----
Update:

I found not only `lib/sass-rails.rb` but other files have wrong file permission.

```
-rw-------+ 1 gussan  staff  184  3 31 15:04 ./rails/generators/sass/scaffold/scaffold_generator.rb
-rw-------+ 1 gussan  staff  185  3 31 15:04 ./rails/generators/scss/scaffold/scaffold_generator.rb
-rw-------+ 1 gussan  staff  227  3 31 15:04 ./sass/rails.rb
-rw-------+ 1 gussan  staff  21  3 31 15:04 ./sass-rails.rb
```

**Comment by ZipoKing** [comment]
Can confirm, in my installation following files are readable only by file owner:
```
pawel(08:45:27):sass-rails-5.0.2$ find . -perm /400
.
./README.md
./lib
./lib/rails
./lib/rails/generators
./lib/rails/generators/sass_scaffold.rb
./lib/rails/generators/scss
./lib/rails/generators/scss/scaffold
./lib/rails/generators/scss/scaffold/scaffold_generator.rb
./lib/rails/generators/scss/assets
./lib/rails/generators/scss/assets/templates
./lib/rails/generators/scss/assets/templates/stylesheet.scss
./lib/rails/generators/scss/assets/assets_generator.rb
./lib/rails/generators/sass
./lib/rails/generators/sass/scaffold
./lib/rails/generators/sass/scaffold/scaffold_generator.rb
./lib/rails/generators/sass/assets
./lib/rails/generators/sass/assets/templates
./lib/rails/generators/sass/assets/templates/stylesheet.sass
./lib/rails/generators/sass/assets/assets_generator.rb
./lib/sass-rails.rb
./lib/sass
./lib/sass/rails.rb
./lib/sass/rails
./lib/sass/rails/version.rb
./lib/sass/rails/cache_store.rb
./lib/sass/rails/importer.rb
./lib/sass/rails/template.rb
./lib/sass/rails/railtie.rb
./lib/sass/rails/logger.rb
./lib/sass/rails/helpers.rb
./MIT-LICENSE
```

**Comment by sue445** [comment]
:+1: 
I'm waiting next version!

**Comment by rafaelfranca** [comment]
:bomb:

**Comment by rafaelfranca** [comment]
Fixed at 5.0.3
